.github/workflows/tests.yaml

```diff
@@ -28,11 +28,11 @@ jobs:
       run: docker build -t civiform-dev --cache-from docker.io/civiform/civiform-dev:latest ./
     - name: Run tests
       run: USE_LOCAL_CIVIFORM=true bin/run-test-ci
-    - name: jacoco-badge-generator
-      uses: cicirello/jacoco-badge-generator@v2
+    - name: Upload coverage to Codecov
```

Comment:
This should probably only run on push to main, otherwise we'll have a lot of uploaded reports, potentially conflicting

Reply:
No it won't.. It is sorted by branch name.. Also this will generate a report comparing the current branch with Main.. so the engineer would know if his/her coverage is pulling down/up the main coverage numbers

Reply:
nice!